app/controllers/v2/content_items_controller.rb

```diff
@@ -41,5 +55,11 @@ def publishing_app
         current_user.app_name
       end
     end
+
+    def return_400_if_no_format
```

Comment:
`params` includes magic rails params like `format` from the path extension, and clashes.